packages/component/src/Utils/AbsoluteTime.js

```diff
@@ -6,14 +6,20 @@ import ScreenReaderText from '../ScreenReaderText';
 import useLocalize from '../hooks/useLocalize';
 import useLocalizeDate from '../hooks/useLocalizeDate';
 
-const AbsoluteTime = ({ value }) => {
+const CalculateAbsoluteTime = value => {
```

Comment:
This seems redundant.... Just use `useLocalizeDate()` in `AbsoluteTime`.

Reply:
It's not. I had to extract CalculateAbsoluteTime because it's being used in other places.

Reply:
Just use `useLocalizeDate` in those other places.